README.md

```diff
@@ -99,6 +105,50 @@ from the Debug > Theme menu. There is not yet a mechanism for including
 custom themes.
 
 
+## CLI
+
+Xi-Mac includes a CLI for opening files directly from the command line.
```

Comment:
I think the name of the actual mac application is (currently) `XiEditor`.

Reply:
Fixed that!